de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Server_Manual.adoc

```diff
@@ -0,0 +1,236 @@
+= Virtual Satellite 4 Core: Server Manual
+:imagesdir: images
+:title-logo-image: images/title/VirtualSatellite_Server.jpg
+:toc:
+:toclevels: 3
+:experimental: 
+
+== Introduction
+
+Virtual Satellite can be run headless as a server that allows you to connect Virtual Satellite to a network, e.g. the Internet.
```

Comment:
"Virtual Satellite can be run headless as a server that allows you to connect Virtual Satellite to a network, e.g. the Internet."
->
"Virtual Satellite can be run headless as a server.
that allows you to connect Virtual Satellite to a network, e.g. the Internet."